server/core/src/actors/v1_read.rs

```diff
@@ -35,6 +38,8 @@ use kanidmd_lib::{
     idm::serviceaccount::ListApiTokenEvent,
 };
 
+use crate::config::LogLevel;
```

Comment:
Why are you importing this? I don't see you using it?

Reply:
Yeah dumb autoimports stuff